### PR TITLE
FIX remove extra param in StimServer

### PR DESCRIPTION
**Pull request by jasmainak**
I wouldn't go for a deprecation as it's more of a bug fix (+ I'm lazy)

### CODE REVIEW

**Comment by agramfort** [comment]
why was this a bug?

**Comment by jasmainak** [comment]
because if you don't set the `ip` param in `StimServer` to be the same as the `ip` param in the `StimClient` it won't work. In my mind, you shouldn't have to provide the `ip` when you instantiate the server. Only the client should be told where the server is

File: examples/realtime/rt_feedback_server.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
 # Instantiating stimulation server
 
 # The with statement is necessary to ensure a clean exit
-with StimServer('localhost', port=4218) as stim_server:
+with StimServer(port=4218) as stim_server:
```
Comment (agramfort) [inline examples/realtime/rt_feedback_server.py]: I still find it weird to pass a port without a server...

Comment (jasmainak) [inline examples/realtime/rt_feedback_server.py]: so what do you suggest?

**Comment by agramfort** [comment]
got it !

can you rebase to make travis happy?

**Comment by jasmainak** [comment]
Travis is happy now

**Pull request merged**